style.css

```diff
@@ -0,0 +1,226 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap');
+
+
+body {
+  font-family: 'Poppins';
+}
+
+header > div {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid lightgray;
+  }
+  
+  nav {
+    width: 600px;
+  }
+  
+  nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  
+  nav a {
+    text-decoration: none;
+    color: black;
+  }
+  
+  .blackHeart {
+    width: 20px;
+    margin-right: 1rem;
+  }
+
+  .underbar {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .npmLogo {
+      height: 20px;
+      padding: 25px 0 25px 0;
+  }
+
+  .lupa {
+    height: 20px;
+    margin-left: 12px;
+  }
+
+  .search-form {
```

Comment:
Various ways of class naming are applied: underbar is lowercase, npmLogo is camel case, and search-form is hyphenated. Please apply one of the naming styles, instead of a mix.

Reply:
Fixed. I was not aware mixing was not best practices, but it makes sense